glide.lock

```diff
@@ -90,6 +90,8 @@ imports:
   - metadata
   - naming
   - peer
+  - stats
+  - tap
```

Comment:
don't think we want these, looks like the grpc update was removed with git add -p and these were left in.